### ISSUE TITLE
Test Centrallix utility modules

### ISSUE THREAD

**Issue by sheesania** [LightSys/centrallix#31]
I'm guessing these are the modules here: https://github.com/LightSys/centrallix/tree/master/centrallix/utility This includes testing the Centrallix version of `stparse.c` (which is presumably different than the Centrallix-Lib version?).

**Comment by gbeeley** [comment]
Yes; the Centrallix version of stparse.c supports expressions in the structure file, whereas the centrallix-lib version only supports constants, because the expression compiler and evaluator is in Centrallix.

There is also an stparse "NE" version in Centrallix / utility / stparse.c.  "NE" stands for "no expressions" but it is also a very lightweight, strings only, version of stparse.  Typically it is just used for creating tree type data structures in RAM.  This could conceivably be moved to centrallix-lib.
